socs/agents/scpi_psu/drivers.py

```diff
@@ -1,6 +1,150 @@
 # Tucker Elleflot
+
+import socket
+import time
+
 from socs.common.prologix_interface import PrologixInterface
 
+# append new model strings as needed
+ONE_CHANNEL_MODELS = ['2280S-60-3', '2280S-32-6']
+THREE_CHANNEL_MODELS = ['2230G-30-1']
+
+
+class ScpiPsuInterface:
+    def __init__(self, ip_address, gpibAddr, port, **kwargs):
+        self.ip_address = ip_address
+        self.gpibAddr = gpibAddr
+        self.port = port
+        self.sock = None
+        self.model = None
+        self.numChannels = 0
```

Comment:
Sorry for the nitpick comment, but conventionally we're trying to use `lower_case_with_underscores` for attributes. Can you replace `numChannels` with `num_channels` everywhere? (And apologies for the confusing use of `gpibAddr` in the old code.)

Reply:
haha no worries. a simple `:%s/numChannels/num_channels/g` did the trick